activemodel/lib/active_model/errors.rb

```diff
@@ -571,6 +571,12 @@ def init_with(coder) # :nodoc:
       add_from_legacy_details_hash(data["details"]) if data.key?("details")
     end
 
+    def inspect # :nodoc:
```

Comment:
I don't think it needs to be `nodoc` as it's public API.
There are lot's of `inspect` overrides visible in the docs:
https://api.rubyonrails.org/?q=inspect

Reply:
@p8  I copied this from [`ActiveRecord::Core#inspect`](https://github.com/rails/rails/blob/212077a972404f19aa520a7792d2b3355f2de741/activerecord/lib/active_record/core.rb#L406) and I assumed there isn't much value in displaying this in the doc as it is pretty arbitrary. WDTY?

Reply:
Yeah, value is now. Fine.

Reply:
I forgot though, could you add a CHANGELOG entry?

Reply:
Looking at the the documentation guides, it states the following about `nodoc`
> To summarize, the Rails team uses :nodoc: to mark publicly visible methods and classes for internal use; changes to the visibility of API should be considered carefully and discussed over a pull request first.

https://guides.rubyonrails.org/api_documentation_guidelines.html#method-visibility

But `inspect` won't be renamed, and removing the override will still make `inspect` public because of the superclass.
🤷  So I agree with you both...